courses/programming-in-haskell/pih-exercises/test/Chapter06Spec.hs

```diff
@@ -1,69 +1,96 @@
--- | Specification for the exercises of Chapter 6.
+-- | Specification for the exercises of Chapter 04.
 
 module Chapter06Spec where
 
-import qualified Chapter06       as C6
+import Chapter06 as C6
+import Test.Hspec (Spec, describe, it, shouldBe)
 
-import           Data.List       (sort)
-import           Data.Proxy
-import           Test.Hspec      (Spec, describe, it, shouldBe)
-import           Test.QuickCheck (Arbitrary (..), Property, property, (.&.),
-                                  (===), (==>))
+spec :: Spec
+spec = do
+  describe "Exercise 1: define a function that" $ do
 
-checkEquivalence :: (Arbitrary a, Show a, Show b, Eq b)
```

Comment:
For some reason the QuickCheck tests seems to have been removed in this PR as well. Could you check what happened?